src/components/CommentSection.jsx

```diff
@@ -1,27 +1,47 @@
-import React from 'react'
-import { View, Button, FlatList, Text } from 'react-native'
+import React, { useEffect, useState } from 'react'
+import { useSelector } from 'react-redux'
+import { View, Button, FlatList, Text, TextInput } from 'react-native'
 import { useRoute } from '@react-navigation/native'
-import styles from '../styles/styles'
+import Comments from '../modules/CommentService'
 
 const CommentSection = () => {
   const route = useRoute()
-  const comments = route.params.post.comments
+  const post = route.params.post
+  const { comments, credentials } = useSelector(state => state)
+  const [newComment, setNewComment] = useState('')
 
-  const SingleComment = ({ content }) => (
-    <View>
-      <Text testID='single-comment'>{content}</Text>
-    </View>
+  useEffect(() => {
+    Comments.index(post.id)
+  }, [])
+
+
+  const SingleComment = ({ content }) => <Text>{content}</Text>
+  const renderComment = ({ item }) => (
+    <SingleComment testID={`comment-text-${item.id}`} content={item.content} />
   )
-  const renderComment = ({ item }) => <SingleComment content={item.content} />
 
   return (
     <View testID='comment-section' name='CommentSection'>
       <FlatList
         data={comments}
         keyExtractor={item => item.id.toString()}
         renderItem={renderComment}
+        extraData={newComment}
```

Comment:
I think we might don't need this. Can you check if the app is working without extraData?

Reply:
It works without, we have now removed it!